homeassistant/components/tplink/quality_scale.yaml

```diff
@@ -0,0 +1,66 @@
+rules:
```

Comment:
Would it make sense to move the `_attr_supported_features` above the constructor? (I am surprised this isn't a rule in ruff)

```py


class TPLinkLightEffectEntity(TPLinkLightEntity):
    """Representation of a TPLink Smart Light Strip."""

    def __init__(
        self,
        device: Device,
        coordinator: TPLinkDataUpdateCoordinator,
        *,
        light_module: Light,
        effect_module: LightEffect,
    ) -> None:
        """Initialize the light strip."""
        self._effect_module = effect_module
        super().__init__(device, coordinator, light_module=light_module)

    _attr_supported_features = LightEntityFeature.TRANSITION | LightEntityFeature.EFFECT
```